### PR TITLE
Add Raven for Sentry

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@pmclanahan r?

File: snippets/wsgi/app.py [hunk 1]
```diff
@@ -14,16 +14,18 @@
...
 
-
 application = get_wsgi_application()
 application = DjangoWhiteNoise(application)
 
 # Add media files
 if settings.MEDIA_ROOT and settings.MEDIA_URL:
     application.add_files(settings.MEDIA_ROOT, prefix=settings.MEDIA_URL)
 
+application = Sentry(application)
```
Comment (pmac) [inline snippets/wsgi/app.py]: Is this part necessary? Basket isn't doing this but it seems to be working fine.

Comment (pmac) [inline snippets/wsgi/app.py]: Ah. I see it [in the docs](https://docs.getsentry.com/hosted/clients/python/integrations/django/#wsgi-middleware). Cool. Perhaps I'll add this to basket too :)

**Comment by pmac** [comment]
r+ 🎆

**Comment by glogiotatidis** [comment]
Thanks!

**Pull request merged**